Sources/TuistLoader/Loaders/ManifestLoader.swift

```diff
@@ -134,12 +134,19 @@ public class ManifestLoader: ManifestLoading {
     // MARK: - Private
 
     private func loadManifest<T: Decodable>(_ manifest: Manifest, at path: AbsolutePath) throws -> T {
-        let manifestPath = path.appending(component: manifest.fileName)
-        guard FileHandler.shared.exists(manifestPath) else {
-            throw ManifestLoaderError.manifestNotFound(manifest, path)
+        var fileNames = [manifest.fileName]
+        if let deprecatedFileName = manifest.deprecatedFileName {
```

Comment:
Currently the manifest doesn't have a `deprecatedFileName` - was this added to support a future case?

Reply:
It's [defined here](https://github.com/tuist/tuist/pull/1083/files#diff-7c4da501bdfa517ff825e22a32bb2a19R11). It's intended to be used for this case. Once we drop the support for the current convention, we can remove that code.